logic/resolvable/Concludable.java

```diff
@@ -233,27 +206,127 @@ boolean unificationSatisfiable(ThingVariable concludableThingVar, ThingVariable
         }
     }
 
-    public static class Relation extends Concludable<RelationConstraint> {
+    private static Set<ValueConstraint<?>> equalsConstraints(Set<ValueConstraint<?>> values) {
+        return Iterators.iterate(values).filter(v -> v.predicate().equals(EQ)).toSet();
+    }
+
+    private static Function<grakn.core.concept.thing.Attribute, Boolean> valueEqualsFunction(ValueConstraint<?> value) {
```

Comment:
This handles equal value equality checking, which we now need not only in Has but also in Isa and Attribute, so it's moved up to sit in the parent class `Concludable`